### PR TITLE
Save post

### PR DESCRIPTION
**Pull request by yongsokheng**

### CODE REVIEW

File: db/migrate/20161005145057_create_save_posts.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateSavePosts < ActiveRecord::Migration[5.0]
+  def change
+    create_table :save_posts do |t|
+      t.references :user, foreign_key: true
```
Comment (NgengChhengkim) [inline db/migrate/20161005145057_create_save_posts.rb]: index: true ?

File: app/models/save_post.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class SavePost < ApplicationRecord
+  belongs_to :user
+  belongs_to :activity
+
```
Comment (NgengChhengkim) [inline app/models/save_post.rb:4]: validate uniq user, activity

File: app/controllers/share_posts_controller.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class SharePostsController < ApplicationController
```
Comment (NgengChhengkim) [inline app/controllers/share_posts_controller.rb]: brer tver ey?

Comment (yongsokheng) [reply app/controllers/share_posts_controller.rb]: save all vea leng mok, ot deng

**Pull request merged**